Formula/openimageio.rb

```diff
@@ -0,0 +1,75 @@
+class Openimageio < Formula
+  desc "Library for reading, processing and writing images"
+  homepage "http://openimageio.org"
+  url "https://github.com/OpenImageIO/oiio/archive/Release-1.8.7.tar.gz"
+  sha256 "ed23ecdd3e4037a411c69a6a681273d01d4a2fbcc223eac86daba77dd03fc5ed"
+  head "https://github.com/OpenImageIO/oiio.git"
+
+  bottle do
+    cellar :any
+    sha256 "77bdd8b9b3e1b1ca6abd7eaa98227b0794b98b43fbacb24348774c4de10a4570" => :high_sierra
+    sha256 "b50f595ad597de8774ac929917d0371d2dca702349e3e4538250dc9fcf0050fd" => :sierra
+    sha256 "02adc761d2d21864e480795a4ef527de7852401770dec5dac31ea99872347001" => :el_capitan
+  end
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+  depends_on "openssl"
+  depends_on "ffmpeg"
+  depends_on "freetype"
+  depends_on "giflib"
+  depends_on "ilmbase"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libtiff"
+  depends_on "opencolorio"
+  depends_on "openexr"
+  depends_on "openjpeg"
+  depends_on "webp"
+  depends_on "cfitsio" => :optional
```

Comment:
These dependencies are really optional and afaik barely used. Do we really want to have all this clutter installed next to openimageio?

Reply:
Do not make them mandatory, but remove them completely, was @ilovezfs' comment

Reply:
Got it.

Reply:
You can require any of the ones you yourself are going to want ;)

Reply:
Why not ask upstream for preferences? @lgritz: Any thoughts on which dependencies shall be mandatory or optional?

Reply:
Mandatory, needed for build and minimal functionality:  boost, ilmbase, libtiff, python (and in the next major release, also pybind11)

Mandatory, because the package is stupid without support for these:  openexr, freetype, giflib, jpeg, libpng

Optional, these are for support of more rarely used formats that most users don't need: ffmpeg, opencolorio, openjpeg, webp, cfitsio

Reply:
Thanks @lgritz! 

So, in conclusion for this PR, `boost`, `ilmbase`, `libtiff`, `python`/`python3`, `openexr`, `freetype`, `giflib`, `jpeg`, `libpng` shall be mandatory without any further discussion needed.

Furthermore, I would like to make `ffmpeg`, `opencolorio`, `openjpeg`, `openssl` and `webp` mandatory, to offer a better *out-of-the-box experience*.

Finally, this means that `cfitsio`, `field3d`, `libraw`, `opencv@2` and `ptex` will get removed.

Reply:
Actually, can I recommend that libraw also be added for good out-of-the-box experience?

I agree that cfitsio, field3d, opencv, and ptex can be removed, they are for specialized uses.

Reply:
Of course. It's your package! You decide.